napps/kytos/of_flow_manager/README.rst

```diff
@@ -32,20 +32,21 @@ Protocol Operation
 
 The endpoints implemented in this napp are presented in the table below.
 
-+----------------------------------------+----------------------------------+------------+
-| Endpoint                               | Description                      | Method     |
-+========================================+==================================+============+
-| ``/flow-manager/dpid/flows``           | Retrieve flows from a specific   | ``GET``    |
-|                                        | Switch                           |            |
-+----------------------------------------+----------------------------------+------------+
-| ``/flow-manager/flows``                | Retrieve all flows               | ``GET``    |
-+----------------------------------------+----------------------------------+------------+
-| ``/flow-manager/dpid/flows-a``         | Add a flow in a specific Switch  | ``POST``   |
-|                                        |                                  |            |
-+----------------------------------------+----------------------------------+------------+
-| ``/flow-manager/dpid/flow_id/flows-d`` | Delete a flow from a specific    | ``DELETE`` |
-|                                        | Switch                           |            |
-+----------------------------------------+----------------------------------+------------+
++------------------------------------------+----------------------------------+------------+
+| Endpoint                                 | Description                      | Method     |
++==========================================+==================================+============+
+| ``/flow-manager/flows[/dpid]``           | Retrieve installed flows from    | ``GET``    |
```

Comment:
@renanrodrigo does this endpoint accepts `/flow-manager/flows` only, without a dpid? If so, it will return all flows from all switches? Is this a good behavior?

If not and the dpid is mandatory, if I am not wrong, the syntax would be:
`/flow-manager/flows/<dpid>`

Is this right @beraldoleal ?

Same comment about the endpoints bellow.

Moreover, for `GET` Method it may be reasonable to accept "all" behavior. But, IMHO, for POST and DELETE methods this is **really bad** and dangerous.